conf/application.conf

```
@@ -84,6 +84,10 @@ auditing {
   }
 }
 
+guidance {
+  url = ""
```

```suggestion
  url = "/"
```
could you set this as "/"?
the breadcrumb component renders as plain text when using an empty string.
So just to make it look correct, can we include the "/"